### DISCUSSION TITLE
this is for interior what about a localized area outside?

### DISCUSSION THREAD

**Discussion by j215chic** [jyjblrd/Mocap-Drones#75] [Q&A] [answered]
hi there
would this work outside and if so what would be the limitations.  VERY interesting project. I'd like to see some kind of perimeter patrol functionality using opensource s/w. A friend has a farm and it would be very helpful for a small drone like this to wander around looking for foxes et al.

**Comment by jyjblrd** [comment]
At that point, the cameras needed to provide positioning for the drone would probably give a better view of whats going on than the drone itself lol. I would look into a system that doesn't require external cameras – eg. visual SLAM

**Comment by j215chic** [comment, marked as answer]
thank you SO much for this wonderful project. Loved it!  I understand but wanted to make sure. LOL that the positioning cameras better solution than any onboard :-) what is visual SLAM?

**Comment by j215chic** [comment]
did some research into visual SLAM ( sorry about that) well done on this project!!!

**Reply by jyjblrd** [reply]
glad to point you in the right direction :)
